libandroid-navigation/src/main/java/org/maplibre/navigation/android/navigation/v5/navigation/NavigationConstants.kt

```diff
@@ -0,0 +1,192 @@
+     * from custom milestones in the [MilestoneEventListener].
+     *
+     * @since 0.8.0
+     */
+    const val BANNER_INSTRUCTION_MILESTONE_ID: Int = 2
+
+    /**
+     * Random integer value used for identifying the navigation notification.
+     *
+     * @since 0.5.0
+     */
+    const val NAVIGATION_NOTIFICATION_ID: Int = 5678
+
+    /**
+     * NavigationLauncher key for storing initial map position in Intent
+     */
+    const val NAVIGATION_VIEW_INITIAL_MAP_POSITION: String = "navigation_view_initial_map_position"
+
+    /**
+     * Text to be shown in AlertView during off-route scenario.
+     */
+    const val REPORT_PROBLEM: String = "Report Problem"
+
+    /**
+     * Duration in which the AlertView is shown with the "Report Problem" text.
+     */
+    const val ALERT_VIEW_PROBLEM_DURATION: Long = 10000
+
+    /**
+     * If a set of light / dark themes been set in [android.content.SharedPreferences]
+     */
+    const val NAVIGATION_VIEW_PREFERENCE_SET_THEME: String = "navigation_view_theme_preference"
+
+    /**
+     * Key for the set light theme in preferences
+     */
+    const val NAVIGATION_VIEW_LIGHT_THEME: String = "navigation_view_light_theme"
+
+    /**
+     * Key for the set dark theme in preferences
+     */
+    const val NAVIGATION_VIEW_DARK_THEME: String = "navigation_view_dark_theme"
+
+    /**
+     * Defines the minimum zoom level of the displayed map.
+     */
+    const val NAVIGATION_MINIMUM_MAP_ZOOM: Double = 7.0
+
+    /**
+     * Maximum duration of the zoom/tilt adjustment animation while tracking.
+     */
+    const val NAVIGATION_MAX_CAMERA_ADJUSTMENT_ANIMATION_DURATION: Long = 1500L
+
+    /**
+     * Minimum duration of the zoom adjustment animation while tracking.
+     */
+    const val NAVIGATION_MIN_CAMERA_ZOOM_ADJUSTMENT_ANIMATION_DURATION: Long = 300L
+
+    /**
+     * Minimum duration of the tilt adjustment animation while tracking.
+     */
+    const val NAVIGATION_MIN_CAMERA_TILT_ADJUSTMENT_ANIMATION_DURATION: Long = 750L
+
+    /**
+     * 125 seconds remaining is considered a low alert level when
+     * navigating along a [org.maplibre.navigation.android.navigation.v5.models.LegStep].
+     *
+     * @since 0.9.0
+     */
+    const val NAVIGATION_LOW_ALERT_DURATION: Int = 125
+
+    /**
+     * 70 seconds remaining is considered a medium alert level when
+     * navigating along a [org.maplibre.navigation.android.navigation.v5.models.LegStep].
+     *
+     * @since 0.9.0
+     */
+    const val NAVIGATION_MEDIUM_ALERT_DURATION: Int = 70
+
+    /**
+     * 15 seconds remaining is considered a high alert level when
+     * navigating along a [org.maplibre.navigation.android.navigation.v5.models.LegStep].
+     *
+     * @since 0.10.1
+     */
+    const val NAVIGATION_HIGH_ALERT_DURATION: Int = 15
+
+    const val NON_NULL_APPLICATION_CONTEXT_REQUIRED: String =
+        "Non-null application context required."
+
+    val WAYNAME_OFFSET: Array<Float> = arrayOf(0.0f, 40.0f)
```

Comment:
Could we add `@JvmField` here to improve the kotlin -> java interop?

Reply:
Added ✅